web/server/models.py

```diff
@@ -554,3 +560,8 @@ class PlanApplyStageTracker(PlanStageTracker):
 
 class PlanCancelStageTracker(PlanStageTracker):
     cancel: t.Optional[PlanStageCancel] = None
+
+
+class FormatFileStatus(BaseModel):
+    path: str
+    status: Status = Status.SUCCESS
```

Comment:
Nit: Does a status of INIT mean anything for formatting a file?

Reply:
It is possible to have `indeterminate` state, where it is not yet clean if file has been formatted not (sinse we are not storing previous state for files)

Reply:
Should the default be INIT then?